connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/ClientE2ETestSuite.scala

```diff
@@ -1161,6 +1161,27 @@ class ClientE2ETestSuite extends RemoteSparkSession with SQLHelper with PrivateM
     val joined = ds1.joinWith(ds2, $"a.value._1" === $"b.value._2", "inner")
     checkSameResult(Seq((Some((2, 3)), Some((1, 2)))), joined)
   }
+
+  test("call_function") {
+    val session: SparkSession = spark
+    import session.implicits._
+    val testData = spark.range(5).repartition(1)
+    try {
+      session.sql("CREATE FUNCTION custom_sum AS 'test.org.apache.spark.sql.MyDoubleSum'")
```

Comment:
I think it's OK to not test persist functions in spark connect, as it seems hard to include the jar containing the UDF. The client-side implementation is quite simple: constructs a small proto message and server-side turns it to `UnresolvedFunction`. Making sure it works for builtin function is good enought.

Reply:
I have fixed the issue. Please wait for the CI

Reply:
Is it really worth it? @zhengruifeng

Reply:
I have communicated with @zhengruifeng and he agreed your opinion. Let's remove the test case for connect.

Reply:
> 

+1, I think we don't need to include this change in this PR